### PR TITLE
Use NerdStorageVault first to find secrets

### PR DESCRIPTION
**Pull request by mbazhlekova**

### CODE REVIEW

**Comment by jpvajda** [comment]
@devfreddy could we get a quick review on this PR?

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "eslint-fix": "eslint nerdlets/ --fix"
   },
   "dependencies": {
+    "@newrelic/nr1-community": "^1.3.0-alpha.2",
```
Comment (jbeveland27) [inline package.json:16]: There's a newer alpha version available: [1.3.0-alpha.5](https://www.npmjs.com/package/@newrelic/nr1-community/v/1.3.0-alpha.5)

I believe it has the same API, but might have some bug fixes. 

CC: @devfreddy

**Review by jbeveland27 — APPROVED** [review]
Looks great @mbazhlekova! Left one small comment to update the `nr1-community` package to the latest alpha version.

**Review by devfreddy — APPROVED** [review]
I'll just echo @jbeveland27 's comment, you should update to the latest alpha and re-test, there was a bug or two resolved. FWIW that version is based off of [this branch](https://github.com/newrelic/nr1-community/commits/feat/nerd-vault).

Otherwise, LGTM 👍

**Pull request merged**